app/package-lock.json

```diff
@@ -8,8 +8,14 @@
       "name": "rc4community",
       "version": "0.3.0",
       "dependencies": {
+        "@rocket.chat/fuselage": "^0.31.3",
```

Comment:
DO NOT commit this file --  get it into your `.gitignore`

Reply:
added it to .gitignore and deleted it from commit